### PR TITLE
[FLOC-1968] License

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-1968

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014, 2015 ClusterHQ
+   Copyright 2014-2015 ClusterHQ
```
Comment (dwgebler) [inline LICENSE]: Seeing as this is a copyright notice, I suspect this should be "ClusterHQ Inc", seeing as that's the actual entity name that owns Flocker.

**Comment by dwgebler** [comment]
There are some flake8 errors in `admin/test_release` - I'm not sure that matters, we have quite a few flake8 errors in our various admin code and we don't care about that in the same way we do for Flocker, but still, nice to fix them in any file you happen to be working on.

Other than the one thing about whether or not to use the full company name in the copyright line (my guess is we should), this looks good and tests fine so I will accept the JIRA issue.

**Comment by adamtheturtle** [comment]
> There are some flake8 errors in admin/test_release - I'm not sure that matters, we have quite a few flake8 errors in our various admin code and we don't care about that in the same way we do for Flocker, but still, nice to fix them in any file you happen to be working on.

We do care and there is a follow up to fix it.

**Comment by exarkun** [comment]
@adamtheturtle @dwgebler 

This change guarantees a failing test suite once a year.  It also uses local time so as to produce different results depending on where it is run. :/

**Comment by dwgebler** [comment]
> This change guarantees a failing test suite once a year.

I thought that was the intention, that we therefore cannot miss when we need to update the file.

Local time I did notice was an issue, but seeing as we're only using the year and probably not ever doing a release in the 12 hour period on New Year's Eve where some parts of the world might consider it a different year to other parts, I didn't think that was worth doing anything about.

**Pull request merged**